.github/workflows/nix.yaml

```diff
@@ -143,21 +152,39 @@ jobs:
           name: macOS
           path: "./packages/macOS/*.pkg"
           if-no-files-found: error
-      - name: Test Installation
+      - name: Test Installation (Ubuntu)
         if: runner.os == 'Linux'
         run: |
           deb_file="$(basename ./packages/debian/tenzir*.deb)"
-          docker build tenzir/services/systemd/test -t installer-test
-          docker run -d --name installer-test \
+          docker build tenzir/services/systemd/test -t installer-test-ubuntu
+          docker run -d --name installer-test-ubuntu \
                  --privileged \
                  --volume=.:/src/tenzir:ro \
                  --volume="./packages/debian":/tmp/packages:ro \
                  --volume=/sys/fs/cgroup:/sys/fs/cgroup:rw \
                  --cgroupns=host \
-                 installer-test
+                 installer-test-ubuntu
+          # TODO: Ideally we would have some way to block until systemd is fully
+          # booted, but sleeping a bit works almost every time.
+          sleep 5
```

Comment:
This section and the waiting time is mostly a copy of the `deb` package installation test. I'm not aware that this test ever failed because it was not long enough.
5 seconds are hardly noticeable, so I don't think we should spend time on optimizing here.